cypress/integration/radioButton.feature

```diff
@@ -0,0 +1,129 @@
+Feature: Radio Button component
+  I want to change Radio Button component properties
+
+  Background: Open Radio Button component page
+    Given I open "Radio Button" component page
+
+  @positive
+  Scenario Outline: Set fieldHelp to <fieldHelp>
+    When I set fieldHelp to "<fieldHelp>"
+    Then fieldHelp on preview is set to "<fieldHelp>"
+    Examples:
+      | fieldHelp               |
+      | Sample text             |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <> |
+
+  @positive
+  Scenario: Check fieldHelpInline checkbox
+    When I check fieldHelpInline checkbox
+    Then fieldHelpInline is enabled
+
+  @positive
+  Scenario: Uncheck fieldHelpInline checkbox
+    When I check fieldHelpInline checkbox
+      And I uncheck fieldHelpInline checkbox
+    Then fieldHelpInline is disabled
+
+  @positive
+  Scenario Outline: Set label to <label>
+    When I set label to "<label>"
+    Then label on preview is "<label>"
+    Examples:
+      | label                   |
+      | Sample text             |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <> |
+
+  @positive
+  Scenario Outline: Check labelInline checkbox
+    When I check labelInline checkbox
+    Then radioButton labelInline is enabled
```

Comment:
Radio Button - it's component name, Capital first letter

Reply:
fixed